crowbar_framework/app/controllers/repositories_controller.rb

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+class RepositoriesController < ApplicationController
+  before_filter :repository_registry
```

Comment:
Thinking about it  after reading the discussion about the registry loading method name, do we really want to always have to load the registry explicitly before accessing any method of the `Repository` class? Why not always load it within the class itself? like a before_filter in `Crowbar::Repository`. Now we are dependent on the explicit call of `Crowbar::Repository.require_registry` otherwise the class is pretty useless...